gemrb/core/Interface.cpp

```diff
@@ -913,15 +916,15 @@ bool Interface::ReadDamageTypeTable()
 	return true;
 }
 
-bool Interface::ReadSoundChannelsTable() const
+bool Interface::ReadSoundChannelsTable()
 {
 	AutoTable tm = gamedata->LoadTable("sndchann");
 	if (!tm) {
 		return false;
 	}
 
+	// reverb is currently ignored
```

Comment:
any plans with this? I know we currently just stored it.
Btw, at least some of the games also provided sndenvrn.2da and sndresrf.2da, which look like a subset of our reverbs.2da.

Reply:
Not right now. Also there is other values around somewhere (I noticed dampening) but I couldn't make a model out of it within a few seconds, so maybe at some later point.